src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -541,28 +540,23 @@ public IEnumerable<PackageInfo> PackageDependencies
         {
             get
             {
-                // Get all packages installed locally
-                var localPackages = GetLocalPackages();
                 // Get set of assemblies used by nodes in this workspace
                 var assembliesUsed = GetAssembliesUsed();
-
-                // Create a dictionary that maps assembly names to the package they are contained in
-                var assemblyPackageDict = new Dictionary<string, PackageInfo>();
-                foreach(var package in localPackages)
-                {
-                    foreach(var assemblyName in package.AssemblyNames)
-                    {
-                        assemblyPackageDict[assemblyName.FullName] = package;
-                    }
-                }
-
-                // Create a list of packages that are used in this workspace
+                
+                // Collect pacakge dependencies of nodes in workspace
                 var packageDependencies = new HashSet<PackageInfo>();
-                foreach(var assemblyName in assembliesUsed)
+                if (CollectingNodePackageDependencies != null)
                 {
-                    if (assemblyPackageDict.ContainsKey(assemblyName.FullName))
+                    foreach (var f in CollectingNodePackageDependencies.GetInvocationList())
                     {
-                        packageDependencies.Add(assemblyPackageDict[assemblyName.FullName]);
+                        if (f.Target is IExtension && (f.Target as IExtension).UniqueId == "FCABC211-D56B-4109-AF18-F434DFE48139")
```

Comment:
Can we get a hold of the pm extension object and query its `uniqueID` instead of hardcoding the ID here? Ideally the unique ID property should have been static.

Reply:
Marking as resolved since we aren't using this approach anymore